crates/bevy_reflect/src/serde/de/processor.rs

```diff
@@ -32,6 +32,10 @@ use crate::{PartialReflect, TypeRegistration, TypeRegistry};
 /// [`DeserializeWithRegistry`] implementation, so this is also useful for
 /// overriding deserialization behavior if you need to do something custom.
 ///
+/// # Serialization equivalent
+///
+/// The serialization equivalent of this is [`ReflectSerializerProcessor`].
+///
```

Comment:
Nit: Can we move this up to above `# Compared to ...` and remove the section header like you did for `ReflectSerializerProcessor`?

Reply:
I did that change already, but didn't save the file :facepalm: Fixed and pushed.